geocoder-arcgis.d.ts

```diff
@@ -0,0 +1,186 @@
+declare class ArcGISAuth {
```

Comment:
Is this file needed?

Reply:
I'm surprised it's not! This types for the `geocoder-arcgis` dependency, whose types are used quite often throughout the code...